error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1246,4 +1247,36 @@ Duration after(StepVerifier.LastStep step, Duration duration) {
       return step.verifyTimeout(duration);
     }
   }
```

Comment:
Right now we have the `StepVerifier`-related rules at the bottom; let's keep that.